generators/client/templates/react/src/test/javascript/jest.conf.js.ejs

```diff
@@ -13,7 +13,7 @@ module.exports = {
   },
   reporters: [
     'default',
-    [ 'jest-junit', { output: './<%= BUILD_DIR %>test-results/jest/TESTS-results.xml' } ]
+    [ 'jest-junit', { output: './<%= BUILD_DIR %>surefire-reports/TESTS-results-jest.xml' } ]
```

Comment:
Could you explain this change? 
It sounds a bit too more maven centric, whereas JHipster allows you to generate an application with graddle, or even without server (app generation with --skip-server allows you to generate a client only application)

Reply:
Do we have to adapt sonar configs in maven/gradle for that?

Reply:
I'd rather have this settings only if we're using maven :
```
<% if (buildTool === 'maven') { %>
[ 'jest-junit', { output: './<%= BUILD_DIR %>surefire-reports/TESTS-results-jest.xml' } ]
<% } else { %>
[ 'jest-junit', { output: './<%= BUILD_DIR %>test-results/jest/TESTS-results.xml' } ]
<% } %>
```

Reply:
Again, don't forget that JHipster allow you to generate a front-only application, with neither maven or gradle.

Reply:
Maybe we should generate surefire reports in the testresults directory, even for Maven.
It would be easier to maintain.

Reply:
I don't think so. We should stick with default configuration as much as possible (as per Policy 1)

Reply:
@PierreBesson that's what I did in the last version.